wallet/src/util/Icons.ts

```diff
@@ -2,6 +2,7 @@ export const icons = {
   IST: 'tokens/IST.png',
   BLD: 'tokens/BLD.svg',
   ATOM: 'tokens/cosmos.svg',
+  IbcATOM: 'tokens/cosmos.svg',
```

Comment:
might as well kludge whatever code reads from agoricNames.vbankAssets to you can avoid this sort of thing

Reply:
Done